src/components/navBar.tsx

```diff
@@ -4,23 +4,25 @@ import Image from "next/image";
 import SiteButton from "./siteButton";
 import Link from "next/link";
 import { useState } from "react";
+import {
+  LinkedinShareButton,
+  FacebookShareButton,
+  TwitterShareButton,
+} from "next-share";
+import { FaCentercode } from "react-icons/fa6";
 
 export default function NavBar() {
-  const [signupClicked, setSignupClicked] = useState(false);
+  const [clickedButton, setClickedButton] = useState("");
 
-  function handleSignupClick() {
-    if (signupClicked === true) {
-      setSignupClicked(false);
-    } else {
-      setSignupClicked(true);
-    }
+  function handleNavButtonClick(e: any) {
+    setClickedButton(clickedButton === e.target.value ? "" : e.target.value);
```

Comment:
this logic un-stores the clicked button in state, and *I think* it's unnecessary unless you have a reason for doing this?

Reply:
it's so you can close the "signup" and "share" options if you're looking around but don't want to choose those options.